tests/Application/config/sylius/1.11/bundles.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
No OS header

Reply:
None of the files in `config` has OS header. Should I add it to all of them?